src/lib/manager_impl.hpp

```diff
@@ -100,6 +100,11 @@ class repl_service_ctx_grpc : public repl_service_ctx {
     ~repl_service_ctx_grpc() override = default;
     std::shared_ptr< mesg_factory > m_mesg_factory;
 
+    NullAsyncResult data_service_request_unidirectional(destination_t to, std::string const& request_name,
+                                                        io_blob_list_t const& cli_buf) override;
+    AsyncResult< sisl::io_blob > data_service_request_bidrectional(destination_t to, std::string const& request_name,
+                                                                   io_blob_list_t const& cli_buf) override;
+
     AsyncResult< sisl::io_blob > data_service_request(std::string const& request_name,
```

Comment:
Do we really need the data_service_request API still?

Reply:
I will remove it after I implement the two new APIs. If I remove it in this PR, I will have to comment out the corresponding unit test code.